package.json

```diff
@@ -8,17 +8,48 @@
     "supabase:stop": "supabase stop",
     "db:migrate": "supabase db reset",
     "db:remote:migrate": "supabase db push",
-    "db:remote:seed": "ts-node supabase/seed-remote.ts"
+    "db:remote:seed": "ts-node supabase/seed-remote.ts",
+    "build": "nest build",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "prepare": "npm run build",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "test": "jest"
   },
   "keywords": [],
   "author": "Black Flower (https://github.com/blkflower)",
   "license": "ISC",
   "devDependencies": {
+    "@supabase/supabase-js": "^1.35.7",
```

Comment:
@PCalape I think this should be in `dependencies` instead of `devDependencies`.

Reply:
Copy on this @arnellebalane. Will apply next commit